pubspec.yaml

```diff
@@ -1,15 +1,18 @@
 name: yaml
-version: 2.1.14
-author: "Dart Team <misc@dartlang.org>"
-homepage: https://github.com/dart-lang/yaml
+version: 2.1.15
+
 description: A parser for YAML.
+author: Dart Team <misc@dartlang.org>
+homepage: https://github.com/dart-lang/yaml
+
 environment:
-  sdk: '>=2.0.0-dev.49.0 <2.0.0'
+  sdk: '>=2.0.0-dev.49.0 <3.0.0'
+  
 dependencies:
-  charcode: "^1.1.0"
-  collection: ">=1.1.0 <2.0.0"
-  string_scanner: ">=0.1.4 <2.0.0"
-  source_span: ">=1.0.0 <2.0.0"
+  charcode: ^1.1.0
+  collection: ^1.14.10
+  string_scanner: ^1.0.2
+  source_span: ^1.4.0
```

Comment:
Why are you narrowing these constraints?

Reply:
(For the reasons I explained yesterday, but I'll add them back)